erpnext/stock/dashboard/item_dashboard.js

```diff
@@ -26,13 +26,14 @@ erpnext.stock.ItemDashboard = Class.extend({
 
 		function handle_move_add(element, action) {
 			let item = unescape(element.attr('data-item'));
+			let uom = cur_frm.doc.stock_uom;
```

Comment:
@nmimsnikhil  dont use cur_frm.doc.stock_uom instead add stock_uom in items list in stock_dashboard.py
add data-stock_uom in stock_dashboard_list.html

let uom = unescape(element.attr('data-stock_uom'));